resources/src/feedback-form/Project/RatingRadio.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 
-const RatingRadio = ({question_id, radioValue, value, setValue, icon, iconChecked}) => {
-  return (
-    <td className='rating-cell'>
-      <label>
-        <input
-          type='radio'
-          name={question_id}
-          checked={value === radioValue}
-          onChange={() => setValue(radioValue)}
-        />
-        <img src={value === radioValue ? iconChecked : icon} />
+const RatingRadio = ({question_id, radioValue, value, setValue, icon, iconChecked, labelText}) => (
```

Comment:
Adding a prop has added quite a bit of complexity to this component, and that complexity is only ever relevant when we're using this component in the Session form.

Could you make a separate RatingRadio component inside the Session folder, and revert this one?